modules/sdk-coin-atom/src/atom.ts

```diff
@@ -226,6 +255,206 @@ export class Atom extends BaseCoin {
     };
   }
 
+  /**
+   * Builds a funds recovery transaction without BitGo
+   * @param {RecoveryOptions} params parameters needed to construct and
+   * (maybe) sign the transaction
+   *
+   * @returns {AtomTx} the serialized transaction hex string and index
+   * of the address being swept
+   */
+  async recover(params: RecoveryOptions): Promise<AtomTx> {
+    // Step 1: Check if params contains the required parameters
+    if (!params.bitgoKey) {
+      throw new Error('missing bitgoKey');
+    }
```

Comment:
Can we please use the user key instead? If we just need the common key chain, the user key should be sufficient. We do not want to couple bitgo's key in a user <> backup recovery.

Reply:
We'll address this in an followup